src/servers/middlewares/auth-router.ts

```diff
@@ -114,4 +120,129 @@ authRouter.post(
   }),
 )
 
+authRouter.post(
+  "/login",
+  wrapAsyncToRunInSpan({
+    namespace: "auth-router",
+    fn: async (req: express.Request, res: express.Response) => {
+      try {
```

Comment:
could we not have a big try catch?

Reply:
if Auth is properly build in app/services layer then there is no need to be in try block